src/storage/src/hummock/sstable_store.rs

```diff
@@ -560,6 +562,18 @@ impl SstableStore {
         }
     }
 
+    pub async fn get_hnsw_graph(
+        &self,
+        graph_file: &HnswGraphFileInfo,
+    ) -> HummockResult<PbHnswGraph> {
+        let todo = 0;
+        let graph_file_path =
+            self.get_object_data_path(HummockObjectId::HnswGraphFile(graph_file.object_id));
+        let encoded_graph = self.store.read(&graph_file_path, ..).await?;
```

Comment:
Should we cache HnswGraphFile in `vector_meta_cache`? Otherwise, we need to read this file from object store for each vector search.

Reply:
Have added to the cache of vector file meta.